src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ CHANGELOG
  * Deprecate `translation:update` command, use `translation:extract` instead
  * Add `PhpStanExtractor` support for the PropertyInfo component
  * Add `cache.adapter.doctrine_dbal` service to replace `cache.adapter.pdo` when a Doctrine DBAL connection is used.
+ * Fix duplicating of paths used by translation debug/update commands.
```

Comment:
should be removed, changelogs are auto-generated from commit messages for bug fixes

Reply:
Done. It was requested by PR template :)

Reply:
The PR template asks to update changelog files only for new features and deprecations. You answered `no` to these rows of the template.

Reply:
Got it. Thanks